### PR TITLE
chromium: update to 87.0.4280.141

### LINKED ISSUE #432: chromium: update to 87.0.4280.141
Stable release announcements since our latest update:
https://chromereleases.googleblog.com/2021/01/stable-channel-update-for-desktop.html

Targets and platforms we need to check:

**chromium-x11:**
- dunfell, clang, MACHINE="intel-core2-32"
- dunfell, clang, MACHINE="intel-corei7-64"
- dunfell, clang, MACHINE="raspberrypi3"
- gatesgarth, MACHINE="raspberrypi3"

**chromium-ozone-wayland:**

- dunfell, clang, MACHINE="intel-corei7-64"
- dunfell, clang, MACHINE="qemuarm64"
- gatesgarth, clang, MACHINE="raspberrypi3"
- dunfell, clang, MACHINE="m3ulcb"

### PR DESCRIPTION
**Pull request by msisov**
Stable release announcements since our latest update:
https://chromereleases.googleblog.com/2021/01/stable-channel-update-for-desktop.html

Test-built with:

* chromium-x11
  * dunfell, clang, MACHINE="intel-core2-32"
  * dunfell, clang, MACHINE="intel-corei7-64"
  * dunfell, clang, MACHINE="raspberrypi3"
  * gatesgarth, MACHINE="raspberrypi3"

* chromium-ozone-wayland:
  * dunfell, clang, MACHINE="intel-corei7-64"
  * dunfell, clang, MACHINE="qemuarm64"
  * gatesgarth, clang, MACHINE="raspberrypi3"
  * dunfell, clang, MACHINE="m3ulcb"

Removed Android Licenses:
  * third_party/android_ndk/NOTICE
  * It's not removed from the project, but rather from the tarball that
    is distributed for Linux builds. Thus, there is no CL that would
    point to that removal.

Fixes #432

### CODE REVIEW

File: recipes-browser/chromium/chromium-gn.inc [hunk 1]
```diff
@@ -18,7 +18,6 @@ SRC_URI += " \
         file://chromium-Move-CharAllocator-definition-to-a-header-f.patch \
         file://remove-enable-dse-memoryssa-command-line-flag.patch \
         file://0003-Fix-sandbox-Aw-snap-for-syscalls-403-and-407.patch \
-        file://0001-Revert-ui-gfx-linux-Remove-2-unnecessary-preprocesso.patch \
```
Comment (rakuco) [inline recipes-browser/chromium/chromium-gn.inc:21]: This file is still present? Did you really want to remove it from the list of patches in this PR?

File: recipes-browser/chromium/files/do-not-specify-march-on-arm.patch [hunk 2]
```diff
@@ -28,5 +28,5 @@ index f7f5f6b3b721..1a1a6756e058 100644
 -        cflags += [ "-mtune=$arm_tune" ]
 -      }
      } else if (current_cpu == "mipsel" && !is_nacl) {
-       ldflags += [ "-wl,--hash-style=sysv" ]
+       ldflags += [ "-Wl,--hash-style=sysv" ]
```
Comment (rakuco) [inline recipes-browser/chromium/files/do-not-specify-march-on-arm.patch:31]: Did the `Wl` -> `wl` change creep into #428 by mistake?

Comment (msisov) [reply recipes-browser/chromium/files/do-not-specify-march-on-arm.patch:31]: yes, I changed that by accident with vim's shortcode that switches to lowercase.

File: recipes-browser/chromium/chromium-gn.inc (hunk 1 above)
Comment (msisov) [reply recipes-browser/chromium/chromium-gn.inc:21]: Yes. It's not needed. I checked as we discussed, so... why should I continue keeping that?

Comment (msisov) [reply recipes-browser/chromium/chromium-gn.inc:21]: removed the file.

**Review by rakuco — APPROVED** [review]
lgtm

**Pull request merged**